07-Customizing-AzSK-for-your-Org/Extending AzSK Module/Readme.md

```diff
@@ -190,10 +191,10 @@ class SubscriptionCore: SVTBase
 	  }
 	}
    ```
-   All the other functions from the class file should be removed.
+> Note: All the other functions from the class file should be removed.
   
-3. 	 If you are modifying the logic for a specific control, then retain the required function; or if you are adding a new control, copy any control function from the base class to the extension class reference.
-	> Note: For a given control in json, the corresponding PowerShell function is provided as value under MethodName property. You can search for that method under the PS script. For eg. In the below case let us assume you want to add a new control that fails if you have more than 2 co-admins. 
+3. 	 If logic for a specific control needs is to be updated, then retain the required function; or if a new control is to be added, copy any control function from the base class to the extension class reference.
```

Comment:
Used "needs to be updated" instead of "needs **is** to be updated"

Reply:
done